docs/modules/ROOT/pages/explanations/decisions/redis_galera.adoc

```diff
@@ -0,0 +1,64 @@
+= Redis and MariaDB/Galera Provisioning
+
+This page combines the decisions for deploying Redis, MariaDB and Galera, as they are already in use for SPKS.
+
+== Problem
+
+We want to re-use proven methods to deploy Redis and MariaDB/Galera.
```

Comment:
Is that the actual problem to solve? That sounds to me like the reasoning _why_. The problem statement should more be like "We need to provide service X on a Kubernetes cluster with the following features: ...". Or something like that...